hooks/__tests__/useNodeJsContributorsApi.test.tsx

```diff
@@ -0,0 +1,124 @@
+import fetchMock from 'jest-fetch-mock';
+import {
+  linkParser,
+  getContributor,
+  fetchRandomContributor,
+  getMaxContributors,
+} from '../useNodeJsContributorsApi';
+
+describe('linkParser', () => {
+  it('should parse the Link header correctly', () => {
+    const linkHeader =
+      '<https://api.github.com/repos/nodejs/node/contributors?per_page=5&page=2>; rel="next", <https://api.github.com/repos/nodejs/node/contributors?per_page=5&page=3>; rel="last"';
+    const parsedLinks = linkParser(linkHeader);
+    expect(parsedLinks).toEqual({
```

Comment:
Instead of asserting the whole object, we can simply assert the parsedLinks.next and parsedLinks.last